packages/tc-schema-sdk/lib/updater.js

```diff
@@ -22,7 +22,8 @@ class SchemaUpdater {
 		schemaDirectory = process.env.TREECREEPER_SCHEMA_DIRECTORY,
 		schemaData,
 	} = {}) {
-		this.updateMode = updateMode;
+		this.updateMode =
+			updateMode || process.env.TREECREEPER_SCHEMA_URL ? 'poll' : 'stale';
```

Comment:
```suggestion
			updateMode || (schemaBaseUrl ? 'poll' : undefined');
```

Not sure if this will pass linting now, but is this more the behaviour we need? Because
- if the user has passed in a schemaBaseUrl manually, we probably still want to have the same behaviour as if they used an env var (at least, that feels more predictable to me)
- if there is a directory/data instead of url, then we don't need any update mode